time stamp
int32 msg_id
float32 theta_ld
float32 theta_l
float32 theta_l_p
float32 theta_m
float32 theta_m_p
float32 epsilon
float32 epsilon_p
---
bool ack